apidoc/Titanium/App/iOS/iOS.yml

```diff
@@ -1252,15 +1252,22 @@ events:
 
       - name: message
         summary: A string containing the localized description of the error.
+        description: |
```

Comment:
Our doc validation doesn't accept `description` values for event properties.

This is because our doc scripts do not read/use/look for descriptions of event properties or method parameters. You'll need to fold the details into the summary string. (Or we need to alter our doc scripts to incorporate descriptions for these when converting to jsduck/json/html/etc)

Reply:
Updated doc!